gslb/gslbutils/gslbutils.go

```diff
@@ -698,3 +698,15 @@ func SetDifference(a, b []string) []string {
 	}
 	return setA.Difference(setB).List()
 }
+
+func SetEqual(a, b []string) bool {
+	setA := sets.NewString()
```

Comment:
`sets.NewString(a...)` can be used here instead of iterating through the slice.

Reply:
Thanks for pointing that out! Fixed!